tools/Common.Netcore.Dependencies.targets

```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Authorization" Version="1.1.18-preview"/>
     <PackageReference Include="Microsoft.Azure.PowerShell.Common" Version="1.1.18-preview"/>
     <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Compute" Version="1.1.18-preview"/>
-    <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Graph.Rbac" Version="1.1.18-preview"/>
+    <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Graph.Rbac" Version="1.0.317-preview"/>
```

Comment:
Please add the entire set of new nuget packages from the common build.  This ensure that any nuget package dependency update is uniformly applied. There end up being 4 files that need to change (3 csproj files and this one). The simplest way is searching the .csproj and .targets files for '1.1.18' in a vscode search of the directory, and replacing with the new version.